simplefs/simplefs.go

```diff
@@ -117,3 +232,238 @@ func (k *SimpleFS) SimpleFSGetOps(_ context.Context) ([]keybase1.OpDescription,
 func (k *SimpleFS) SimpleFSWait(_ context.Context, opid keybase1.OpID) error {
 	return errors.New("not implemented")
 }
+
+// remotePath decodes a remote path for us.
+func remotePath(path keybase1.Path) (ps []string, public bool, err error) {
+	pt, err := path.PathType()
+	if err != nil {
+		return nil, false, err
+	}
+	if pt != keybase1.PathType_KBFS {
+		return nil, false, errOnlyRemotePathSupported
+	}
+	raw := path.Kbfs()
+	if raw != `` && raw[0] == '/' {
```

Comment:
I think the way I have CLI now is that on Windows, the incoming KBFS path can start with the mounted drive letter instead of "keybase" - but we could normalize that to always start with "/keybase" if you want. Also, speaking of slashes, shouldn't we use os.PathSeparator instead of slash? or does that always have to be normalized to slashes too?